core/application/utils/messageConstants.ts

```diff
@@ -0,0 +1,16 @@
+export const EMAIL_VERIFICATION_TITLE = 'Welcome to Blood Connect!'
+export const PASSWORD_RESET_TITLE = 'Reset your password for Blood Connect'
+
+export const getEmailVerificationContent = (userName: string, securityCode: string): string => `
+  Hello ${userName},<br/><br/>
+  Welcome! Please verify your email using the following code: ${securityCode}.<br/><br/>
+  Thanks!
+`
+
+export const getPasswordResetContent = (userName: string, securityCode: string): string => `
+  Hello ${userName},<br/><br/>
```

Comment:
when i said move these to a const folder, I didn't mean it should be function. Instead,
messageConsts.

const EMAIL_VERIFICATION_CONTENT = "Hello {0}, code{1}"


you can then expose a function if you want that does
EMAIL_VERIFICATION_CONTENT.format()

that would make managing tabs and indents easier.